pep-0676.rst

```diff
@@ -74,38 +77,42 @@ There are several requests for additional features in reading PEPs, such as:
 * support for SVG images [3]_
 * typographic quotation marks [4]_
 * additional footer information [5]_
-* intersphinx functionality [6]_
+* Intersphinx functionality [6]_
 
 These are "easy wins" from this proposal, and would serve to improve the
 quality-of-life for consumers of PEPs (including reviewers and writers). For
-example, the reference implementation no longer requires a scheduled render
+example, the new reference implementation no longer requires a scheduled render
```

Comment:
It may technically be redundant so long as the reader carefully reads the rest of the sentence ("no longer requires"), but I'd argue it adds a bit of clarity. However, examining it in context, it isn't so much as I thought.

Reply:
It may technically be redundant so long as the reader carefully reads the rest of the sentence ("no longer requires"), but I'd argue it adds a bit of clarity. However, examining it in context, it isn't so much as I thought.